faq/faq.md

```diff
@@ -0,0 +1,351 @@
+If the work was licensed differently from your project, you should verify
+whether the licenses are compatible, and add the new licenses to your project.
+
+## Can I copy a work that has no copyright notice or license? {#no-copyright-license}
+
+Before you proceed, always first make sure that you can find the copyright and
+licensing information elsewhere. Some projects only include this information in
+the root directory or in their README file.
+
+If you can find no copyright notice, then that is no problem. You can add the
+copyright notice yourself.
+
+If the work has no license, then that means that you do not have the right to
+copy it. If you believe that this is a mistake and the author clearly meant for
+you to be able to copy this work, you should contact the author and ask them to
+license their work.
+
+## Where else do I put my license information? {#where-else}
+
+README
+
+## What is a copyright holder, and what is an author? {#copyright-holder-author}
+
+## I changed a single line of code. Should I add an SPDX-Copyright tag with my name? {#when-copyright}
+
+The core question here is: At what point should I consider myself a copyright
+holder over a file? This is up to your discretion. It might help to be
+consistent and add the tag to every file you touch, but it is perhaps more
+valuable to reach a consensus about this within your project.
+
+## How do I deal with a file that has been edited by many people? {#many-copyright-statements}
+
+Some files are edited by many people and would have an extremely long list of
+copyright holders in the header. This may be aesthetically unpleasing, but is
+not incorrect.
+
+If you would rather not deal with having so many copyright notices, some
+projects such as Chromium circumvent this problem by using "Copyright (c) 2013
+The Chromium Authors" as their copyright tag. You may consider doing this, but
+you should keep a list of authors in your project.
+
+## Why can't I just use version control to record copyright? {#vcs-copyright}
+
+In [a previous question](#copyright-holder-author), we distinguished between
+copyright holders and authors, which are not always the same. Version control
+typically only records authorship, which makes it unsuitable for the task of
+recording copyright.
+
+Another obstacle is that version control history may contain errors, and fixing
+such an error would require rewriting the history, causing all contributors to
+have to re-download the new trunk.
+
+## Can I use DEP5 to declare licensing and copyright? {#dep5}
+
+See next question.
+
+## Can I bulk-license whole directories? {#bulk-license}
+
+DEP5
+
+## Can I license only a part of a file as being under a different license? {#partial-license}
+
+The short answer is that yes, you can, but no, there is no standard way for
+REUSE to recognize this. If you have a small segment of a file that is licensed
+differently, you should list that license under a separate
+`SPDX-License-Identifier` tag in the header.
```

Comment:
You can already use the SPDX annotation to state that code in this file is under several licenses.

Let’s say someone contributes an snippet under `MIT` into a file that is otherwise under `GPL-2.0-or-later`.

On a file-level you can already annotate that correctly with:

```
SPDX-License-Identifier: GPL-2.0-or-later AND MIT
```

This already tells you that there is code under GPL-2.0-or-later and also code under MIT in this file.

In addition to that, and optionally, you may also use the SPDX ID in the snippet itself – e.g.:

```
[…]
# This piece of code was copied from project X <https://project-x.example>.
# SPDX-Copyright: © 1997 Project X contributors <https://project-x.example>
# SPDX-License-Identifier: MIT
[actual code here]
# end of project X snippet
[…]
```

Reply:
On using `GPL-2.0-or-later AND MIT`, I gave an example on this in another comment concerning `MIT AND 0BSD`.

On using the SPDX tag in the snippet itself: This is technically not conformant with the spec. The spec mandates that the tags be in the header. I'm not sure we (currently) want to deal with snippet-level complexity.